### PR TITLE
Update vnet-deployment.md

### PR DESCRIPTION
**Pull request by bwatts64**
Clarifying two things that I'm seeing customers get confused about
1) Accessing the public endpoint from outside of Azure requires a rule to be added to the NSG
2) Our current Private Link/Private Endpoint solution doesn't work if you provision Dev/Test because the load balancers are basic instead of standard

### CODE REVIEW

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by shsagir** [comment]
@name Thanks for contacting us with feedback about our product documentation. Depending on the complexity, it could take from a few business days to a few weeks to look further into your question, issue, or suggestion. We'll contact you if we need further information, and you'll receive an email message each time we "@mention" you in a comment.

File: data-explorer/vnet-deployment.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ You can access your Azure Data Explorer cluster using the following IP addresses
 * **Private IP**: Used for accessing the cluster inside the VNet.
 * **Public IP**: Used for accessing the cluster from outside the VNet for management and monitoring, and as a source address for outbound connections started from the cluster.
 
+> [!IMPORTANT]
+> The default NSG rules will block access to the Public IP outside of Azure. In order to reach the Public Endpoint you need to add an exception in the NSG for your Public IP address or addresses.
```
Comment (basaba) [inline data-explorer/vnet-deployment.md:32]: outside of the VNet

**Review by basaba — CHANGES_REQUESTED** [review]

**Comment by PRMerger18** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by shsagir** [comment]
#sign-off
Reviewed by docs. PR Review team, please publish. Tx

**Comment by PRMerger12** [comment]
@shsagir: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger14** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**